pkg/collector/corechecks/oracle/oracle_t.go

```diff
@@ -0,0 +1,313 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This should be named `oracle_test.go`.

Reply:
The naming `oracle_t.go` is intentional because we haven't automated spinning up the docker image that is required for the tests yet. Consequently, renaming the file to `oracle_test.go` would cause CI tests to fail. For now, we are running the tests locally and are renaming the file to `oracle_t.go` before pushing changes to repo. 

We prioritised building new functionality over completing the CI pipeline in order seize the opportunity  to generate more revenue, and we will close this gap as soon as we create a minimal viable product that we can sell. 

Is that trade off acceptable to you?

Reply:
We should leverage the build tags system already implemented in the repo.

You can add a custom build tag for that file that will prevent it from being run (ex: `//go:build oracle` or something similar). Since this tag is not in [the tag list used to run unit tests](https://github.com/DataDog/datadog-agent/blob/main/tasks/build_tags.py#L154) the file will be ignored.

But, you will be able to run your oracles test with `inv test --build-include=oracle`.

Reply:
Added the tag on the top of the file. This seems though to break the copyright linter.